codebase/src/java/disco/org/openlvc/disco/connection/rpr/model/InteractionClass.java

```diff
@@ -97,8 +97,11 @@ public void addParameter( ParameterClass parameter )
 		// make sure we don't already have one by this name
 		if( parameters.containsKey(parameter.getName()) )
 		{
-			throw new DiscoException( "Add Parameter Failed: Class [%s] already has parameter with name [%s]",
-			                          name, parameter.getName() );
+			// FIXME - we'll assume that this parameter has the same definition - which makes this fine
+			// but if it were defined any differently we should error out.
+			return;
```

Comment:
Ideally this should have equivalency checks on the parameters as per portico. Are we just treating this current change as a stop-gap to get current work out of the way?

Reply:
Yep exactly. I think the way DISCO is doing this merging would need to change somewhat fundamentally for us to be able to do such equivalency checks - as far as I can tell - the only info we have about parameters at this point is the name, so would need to rework so that we have the full definition of the existing and the incoming parameters.

Reply:
No, I lie - Parameters are also needed (for handle lookup). 

This will need to expand to just loop over all params and add them to the set of cached parameters. If the RTI allows the set of FOM modules then you know they should comply with any HLA merging rules, so there should be no issue with just cumulatively adding to the store of parameters.